setup.py

```diff
@@ -7,8 +7,22 @@
 from datetime import datetime
 from glob import glob
 
-from distutils.command.build import build
-from distutils.core import Command
+try:
+    import multiprocessing
+except ImportError:
+    multiprocessing = None
+
+from distutils.command.clean import clean
+from setuptools import Command
+from setuptools.command.build_ext import build_ext
+from setuptools import setup, find_packages, Extension
+from setuptools.command.sdist import sdist
+
+try:
+    import Cython
+    from Cython.Build import cythonize
+except ImportError:
+    Cython = None
```

Comment:
Hmm, even better maybe: if you only use cythonize, do the None trick with that name.

Reply:
I thought 'Cython' could be clearer, but I suppose this is cleaner. done.